armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -1342,64 +1391,81 @@ def getNextAvailableXsTypes(self, howMany=1, excludedXSTypes=None):
             )
         return availableXsTypes[:howMany]
 
-    def _getUnrepresentedBlocks(self, blockCollectionsByXsGroup):
```

Comment:
`Unrepresented` was used for 3 things within the code which was confusing to read. i renamed the other 2

(this and infinitely dilute nuclides) to help clarify